assets/js/react/features/ErrorHandler.js

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect } from 'react';
+import { isNil } from 'ramda';
+import { useDispatch, useSelector } from 'react-redux';
+import { useIntl } from 'react-intl';
+import { useNotification } from '../hooks';
+import { cleanError } from '../actions/Error';
+
+const ErrorHandler = () => {
+  const dispatch = useDispatch();
+  const { formatMessage } = useIntl();
+  const { error } = useSelector((state) => state.error);
+  const { onError, onWarning } = useNotification();
+
+  useEffect(() => {
+    if (!isNil(error)) {
+      const { message, type } = error;
+      const errorMessage = isNil(message)
+        ? formatMessage({ id: 'unexpected error' })
+        : message;
+      switch (Number(type)) {
+        case 400:
+        case 404:
+        case 500:
```

Comment:
Tu affiches la même erreur pour 400 / 404 / 500 si j'ai bien compris ce code ?

Reply:
Oui
A voir si il faut differencier

Reply:
Je pense que c'est mieux oui. Même si ça reste des messages vagues et généraux, ça aide à comprendre l'erreur (surtout la 404). 

Mes propositions :

400 : `You request was not understood.` ou `Your request is invalid.`
404 : `The resource your are looking for was not found.`